src/providers/gleam/mod.rs

```diff
@@ -96,8 +96,6 @@ impl GleamProvider {
     }
 
     fn get_start(&self, _app: &App, _env: &Environment) -> StartPhase {
-        let mut phase = StartPhase::new("./build/erlang-shipment/entrypoint.sh run");
-        phase.only_include_files = Some(vec!["build/erlang-shipment".into()]);
-        phase
+        StartPhase::new("./build/erlang-shipment/entrypoint.sh run")
```

Comment:
[failed build](https://github.com/railwayapp/nixpacks/actions/runs/12618956113/job/35163065425?pr=1252)

the `phase.only_include_files = Some(vec!["build/erlang-shipment".into()]);` was never working to begin with (since it was fixed in this PR). We can remove the line to keep the previous behaviour.

![CleanShot 2025-01-05 at 20 55 45@2x](https://github.com/user-attachments/assets/023a3fd2-06b4-46bb-a2de-d3abd7762ee0)